web/Controllers/PropertyController.cs

```diff
@@ -140,6 +140,24 @@ public ActionResult AddNewProperty_Cancel()
         // Go to View Properties page
         return RedirectToAction("ViewProperties", "Landlord");
     }
+    
+    [Authorize(Roles = "Landlord")]
+    public ActionResult DeleteProperty(int propertyId)
+    {
+        PropertyDbModel? propDB = _propertyService.GetPropertyByPropertyId(propertyId);

```

Comment:
Need to check that it's this landlord's property! `[Authorize(Roles = "Landlord")]` just checks that they are a landlord of some kind. It would be nice if admins can delete any property as well.